node-launchpad/src/config.rs

```diff
@@ -52,6 +78,8 @@ pub async fn configure_winsw() -> Result<()> {
 pub struct AppData {
     pub discord_username: String,
     pub nodes_to_start: usize,
+    pub storage_mountpoint: Option<String>,
```

Comment:
Can we make this into `pub storage_mountpoint: Option<PathBuf>`. Less error as people might have weird chars in their Path & converting to string and back to PathBuf can fail.